src/main/java/codeu/model/data/Notification.java

```diff
@@ -0,0 +1,59 @@
+package codeu.model.data;
+
+import codeu.model.data.User;
+import codeu.model.data.Conversation;
+import codeu.model.data.Message;
+import java.time.Instant;
+import java.util.UUID;
+import java.util.List;
+import java.util.ArrayList;
+import codeu.model.data.Event;
+import codeu.model.store.basic.EventStore;
+
+public class Notification{
+  private  List<String> notification;
+  private User receiver;
+  private User sender;
+  private UUID id;
+  private Event theNotification;
+  private boolean seenNotifcation;
+
+  public Notification(UUID id, User receiver, User sender, Event theNotification){
+      this.receiver = receiver;
+      this.sender = sender;
+      this.id = id;
+      this.theNotification = theNotification;
+  }
+
+  public User getReceiver(){
+    return receiver;
+  }
+
+  public User getSender(){
+    return sender;
+  }
+
+  public UUID getId(){
+    return id;
+  }
+
+  public List<String> getNotification(){
```

Comment:
is there a reason you need these as a list? Maybe you could have getSender(), getReceiver() ... methods that will get that information for you from the notification object right.